src/lib_ccx/debug_def.h

```diff
@@ -0,0 +1,11 @@
+#ifndef _DEBUG_DEF_H_
```

Comment:
This is just a helper for debugging. You could remove this file, and all references to `LOG_DEBUG`... But they are useful for debugging (with existing tests)